imagery-ol/mapSourceProviders/open-layers-GEE-source-provider.ts

```diff
@@ -44,8 +45,9 @@ export class OpenLayerGEESourceProvider extends OpenLayersMapSourceProvider {
 		return Promise.resolve(tileLayer);
 	}
 
-	createSource(metaData: IMapSettings): any {
+	async createSource(metaData: IMapSettings): Promise<any> {
 		const config = { ...this.config, ...metaData.data.config };
+		this.layerData = await this.getLayersData(config.serverUrl);
```

Comment:
You can remove those 2 lines and the async keyword and instead, inside the create method you can always execute the getLayersData method and remove the if